notebook/i18n/README.md

```diff
@@ -53,6 +53,11 @@ pybabel extract -F babel_nbjs.cfg -o nbjs.pot --no-wrap --project Jupyter .
 
 After this is complete you have 3 `.pot` files that you can give to a translator for your favorite language.
 
+Finding and translating the `.pot` files and the `.json` file can be done with the following repository,
+[GitHub Repository Link](https://github.com/berendjan/Jupyter-Notebook-Azure-Translator.git),
+however this requires an Azure account which provides a free translations API up to 2M characters, this
+is more than enough for the translation of the 3 `.pot` files.
+
```

Comment:
This paragraph feels a little odd to me given that it minimally requires a free "trial" account.  I agree that its useful to know about, I just don't know if this kind of thing should appear in the README.

Reply:
Please check the below commit, which notes is is only a possible starting point. Is that better?

Reply:
The update looks great -thank you.